packit_service/worker/mixin.py

```diff
@@ -191,6 +194,9 @@ def packit_api(self):
                 self.service_config,
                 self.job_config,
                 upstream_local_project=self.local_project,
+                dist_git_clone_path=Path(
+                    self.service_config.command_handler_work_dir
+                ).joinpath(SANDCASTLE_DG_REPO_DIR),
```

Comment:
Not sure if this is nicer since it doesn't fit in a single line, what do you think?

```suggestion
                dist_git_clone_path=Path(self.service_config.command_handler_work_dir)
                / SANDCASTLE_DG_REPO_DIR,
```